int32 id
int32 redu_ratio
---
bool redu_ratiograb
